examples/functions/functions.bal

```diff
@@ -5,7 +5,7 @@ function printValue(string value) {
     io:println(value);
 }
 
-// This function takes two int values and return their sum as an integer.
+// This function takes two int values and returns their sum as an integer.
```

Comment:
Shall we have 'int' between backticks? I notice we have backticks in other places so let's be consistent and add backticks for all instances.